17.03/Dockerfile

```diff
@@ -1,30 +1,30 @@
 FROM alpine:3.5
 
 RUN apk add --no-cache \
-		ca-certificates \
-		curl \
-		openssl
+		ca-certificates
 
-ENV DOCKER_BUCKET get.docker.com
+ENV DOCKER_CHANNEL stable
 ENV DOCKER_VERSION 17.03.1-ce
-ENV DOCKER_SHA256_x86_64 820d13b5699b5df63f7032c8517a5f118a44e2be548dd03271a86656a544af55
-ENV DOCKER_SHA256_armel f05c733c22915b0c26d8b0390b26a026b33aaf7593d15475a6f86e1bbe1ddbe2
+# TODO ENV DOCKER_SHA256
```

Comment:
https://download.docker.com doesn't have checksums or ARM builds for static binary releases